src/lib/pages/backoffice/Series/SeriesDetails/SeriesContent/SeriesContent.js

```diff
@@ -30,20 +33,50 @@ export class SeriesContent extends Component {
         )}
         {!_isEmpty(series.metadata.tags) && (
           <>
-            <Divider />
+            {!_isEmpty(series.metadata.abstract) ? <Divider /> : null}
             <Header as="h3">Tags</Header>
             <LiteratureTags size="mini" tags={series.metadata.tags} />
           </>
         )}
 
         {!_isEmpty(series.metadata.keywords) && (
           <>
-            <Divider />
+            {!_isEmpty(series.metadata.abstract) ||
+            !_isEmpty(series.metadata.tags) ? (
+              <Divider />
+            ) : null}
             <Header as="h3">Keywords</Header>
             <br />
             <LiteratureKeywords keywords={series.metadata.keywords} />
           </>
         )}
+
+        {(!_isEmpty(series.metadata.physical_volumes) ||
```

Comment:
I am totally lost with all these `!_isEmpty` :)
Why checking for `keywords` or `abstract` to display `electronic_volumes_description` and `physical_volumes`?

Reply:
I am checking for `keywords`, `abstract` or `tags` to display a divider juts before the `volumes description `section (not to display the section). It's a bit ugly so maybe one solution could be to create components for each section and add a prop called `divider`/`hasPreviousSection` and this way it would be more readable. WDYT?

Reply:
I think have fixed the isEmpty on the document details page on the frontsite in one of my PR recently. You could take a look and do similar solution